CHANGES.txt

```diff
@@ -1,5 +1,6 @@
 1.0.0
 -----
+ * Move root project into a subproject in gradle (CASSANDRASC-145)
```

Comment:
I feel that the ticket does not need to be mentioned. The patch only changes the build files. Per https://cassandra.apache.org/_/development/patches.html
> Please note that only user-impacting items [should](https://lists.apache.org/thread.html/rde1128131a621e43b0a9c88778398c053a234da0f4c654b82dcbbe0e%40%3Cdev.cassandra.apache.org%3E) be listed in CHANGES.txt. If you fix a test that does not affect users and does not require changes in runtime code, then no CHANGES.txt entry is necessary.